pkg/operator/controller/controller.go

```diff
@@ -40,18 +40,18 @@ type Controller interface {
 }
 
 // Builder is a struct, that when complete, registers the passed reconciler with the manager stored
-// insider of the builder. Typed reference implementations, see controllerruntime.Builder
+// insider of the builder. Typed reference implementations, see ctrl.Builder
 type Builder interface {
 	// Complete builds a builder by registering the Reconciler with the manager
 	Complete(Reconciler) error
 }
 
-// Adapter adapts a controllerruntime.Builder into the Builder interface
+// Adapter adapts a ctrl.Builder into the Builder interface
 type Adapter struct {
-	builder *controllerruntime.Builder
+	builder *ctrl.Builder
 }
 
-func Adapt(builder *controllerruntime.Builder) Builder {
+func Adapt(builder *ctrl.Builder) Builder {
 	return &Adapter{
 		builder: builder,
 	}
```

Comment:
nit(style): seems we use controllerruntime everywhere else. I like ctrl more because its shorter and logging is one of those things you type out a lot. Thoughts on picking either `ctrl` or `controllerruntime` to have a uniform reference throughout the codebase?